@eugene-okulik вопрос по ошибке линтера - ./homework/mikhail_luzan/Homework_6/task_1.py:2:119: W504 line break after binary operator

По видео из первого урока, мы говорили про добавление знака '+' при переносе текста для лучшей читабельности.
А здесь линтер считает это ошибкой?

--- a/homework/mikhail_luzan/Homework_6/task_1.py
+++ b/homework/mikhail_luzan/Homework_6/task_1.py
@@ -1,6 +1,6 @@
 text = (
-        'Etiam tincidunt neque erat, quis molestie enim imperdiet vel. Integer urna nisl, facilisis vitae semper at,' +
-        ' dignissim vitae libero'
+    'Etiam tincidunt neque erat, quis molestie enim imperdiet vel. Integer urna nisl, facilisis vitae semper at,'
+    + ' dignissim vitae libero'
 )
 words = text.split()
 final_text = []

